java/maven/src/org/netbeans/modules/maven/execute/MavenProxySupport.java

```diff
@@ -935,7 +949,11 @@ public int next() throws XmlPullParserException, IOException {
         private LineAndColumn startPos() {
             int ln = getLineNumber();
             int col = getColumnNumber();
-            col -= (posEnd - posStart);
+            try {
+                col -= (posEndField.getInt(this) - posStartField.getInt(this));
+            } catch (IllegalAccessException ex) {
+                Exceptions.printStackTrace(ex);
+            }
```

Comment:
this is ugly, those fields are now private since https://github.com/codehaus-plexus/plexus-utils/commit/797cab4d0fb5ce88ef6863cc504f929f6978ff95

Reply:
@sdedic question two: how do I trigger this code path to check if this still works?

I think we have two options here for the mid term: find a different solution for this, or talk to upstream and make the two fields accessible again so that we can remove this hack.

Reply:
This whole thing is triggered when:
- maven `settings.xml` specifies a proxy
- the system does not use a proxy (so no http*_proxy env var or reachable PAC file)
- a maven build or priming build is triggered, or other request in 'online' mode.

I will try to create a simple unittest to verify the settings.xml parsing produces the expected results.